cmd/gcoreclient/main.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/G-Core/gcorelabscloud-go/client/keystones/v1/keystones"
 	"github.com/G-Core/gcorelabscloud-go/client/l7policies/v1/l7policies"
 	"github.com/G-Core/gcorelabscloud-go/client/lifecyclepolicy/v1/lifecyclepolicy"
-	"github.com/G-Core/gcorelabscloud-go/client/limits/v1/limits"
+	"github.com/G-Core/gcorelabscloud-go/client/limits/v2/limits"
```

Comment:
It seems that you completely replaced v1 endpoint with v2.

Do wee need to preserve v1 for backwards compatibility?

Reply:
If you are sure that we don't need to save both v1 and v2, I think you also need to delete the v1 files `v1/limits/*`. Because they won't be used in this case.

Reply:
yes, i've plan delete v1 in future PRs after add all v2 hendler actions (read/create/update). It stay because i want see code v1 while adding v2. But disabled becase v1 disabled in cloud-api and will not revert.

Reply:
ok, got it

don't forget to delete then :)

Reply:
Was removed in the current PR and added other features.